### PR TITLE
Create queue_submit_flow.md

### PR DESCRIPTION
**Pull request by niklassiemer**
I wrote this for @ligerzero-ai to guide to job-submission-flow with pyiron. I think this is valuable to have, however, I do not know a good place to store it...

### CODE REVIEW

**Comment by jan-janssen** [comment]
This seems to be similar to https://pyiron.readthedocs.io/en/latest/source/installation.html#hpc-cluster-with-postgresql-database-and-jupyterhub

**Comment by niklassiemer** [comment]
Yes, only a bit more into the code and less into install/configuration. Something, I would rather show to developers than to users... 

However, we will most probably rework the installation documentation (especially sort it/provide links to the specific installation for the different usage scenarios) and then the in-depth info might fit in somewhere. 

This PR also does not add/list the document in any  file building the documentation. Therefore it is quite hidden atm.

**Comment by jan-janssen** [comment]
At least we should have some links to the existing documentation.

File: docs/source/queue_submit_flow.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+#SBATCH --get-user-env=L
+
+pwd;
+echo Hostname: `hostname`
+echo Date: `date`
+echo JobID: $SLURM_JOB_ID
+
+{{command}}
+```
+(see https://github.com/eisenforschung/pyiron-resources-mpie/blob/master/queues/cmti.sh)
```
Comment (jan-janssen) [inline docs/source/queue_submit_flow.md]: This is only accessible for MPIE - better link to https://github.com/pyiron/pysqa/tree/master/tests/config/slurm/slurm.sh

File: docs/source/queue_submit_flow.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+then
+        unset I_MPI_HYDRA_BOOTSTRAP;
+        unset I_MPI_PMI_LIBRARY;
+        mpiexec -n 1 vasp_std
+else
+        srun -n 1 --exclusive --mem-per-cpu=0 -m block:block,Pack vasp_std
+fi
+```
+
+https://github.com/eisenforschung/pyiron-resources-mpie/blob/master/vasp/bin/run_vasp_5.4.4.sh
```
Comment (jan-janssen) [inline docs/source/queue_submit_flow.md]: Again only accessible for MPIE, better link to https://github.com/pyiron/pyiron-resources/blob/master/lammps/bin/run_lammps_2020.03.03_mpi.sh

**Review by jan-janssen — APPROVED** [review]
Looks good to me

**Comment by niklassiemer** [comment]
Let's see where this ends up if we rewrite some of the documentation... Thanks for the review!

**Pull request merged**